src/amber/cli/templates/app/shard.yml.ecr

```diff
@@ -18,8 +18,8 @@ targets:
 dependencies:
   amber:
     github: amberframework/amber
-    branch: master
-    #version: <%= Amber::VERSION %>
+    version: <%= Amber::VERSION %>
+    #branch: master
```

Comment:
I take it, we will uncomment this later?

Reply:
That should be changed before merge in. Let me do that now.